apps/web/src/components/patient-documents/PatientDocumentUpload.tsx

```
@@ -0,0 +1,687 @@
+  const totalDocuments = existingDocuments.length + uploadQueue.size;
+  const canUpload = !disabled && totalDocuments < maxFiles;
+
+  return (
+    <DndContext
+      onDragStart={({ active }) => setActiveId(active.id)}
+      onDragEnd={() => setActiveId(null)}
+    >
+      <div
+        className={cn('space-y-6', className)}
+        role='region'
+        aria-label='Upload de documentos do paciente'
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<section>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)